docs/2_3_common_states.adoc

```diff
@@ -880,12 +892,22 @@ Getting variables might trigger <<selectiv-computation,computations>>.
 _[If <<Terminated>> is entered because of an `fmi3Error` return value, retrieved values should only be used for debugging purposes.]_
 
 Function <<fmi3GetContinuousStateDerivatives>>::
+Not allowed in Co-Simulation and Scheduled Execution.
+
 Function <<fmi3GetContinuousStates>>::
+Not allowed in Co-Simulation and Scheduled Execution.
+
 Function <<fmi3GetNominalsOfContinuousStates>>::
+Not allowed in Co-Simulation and Scheduled Execution.
+
 Function <<fmi3GetEventIndicators>>::
+Not allowed in Co-Simulation and Scheduled Execution.
+
 Function <<fmi3GetDirectionalDerivative>>::
+No restrictions.
+
 Function <<fmi3GetAdjointDerivative>>::
 No restrictions.
 
 Function <<fmi3GetOutputDerivatives>>::
-Not allowed in Model Exchange.
+Not allowed in Co-Simulation and Scheduled Execution.
```

Comment:
Not allowed in Model Exchange and Scheduled Execution

Reply:
I changed it as suggested, now consistend with "Getting Derivatives of Continuous Outputs"